modules/machine-health-checks-resource.adoc

```diff
@@ -7,28 +7,60 @@
 
 The MachineHealthCheck resource resembles the following YAML file:
 
+// Olde sample
+// .MachineHealthCheck
+// [source,yaml]
+// ----
+// apiVersion: healthchecking.openshift.io/v1alpha1
+// kind: MachineHealthCheck
+// metadata:
+//  name: example <1>
+//  namespace: openshift-machine-api
+// Spec:
+//   Selector:
+//     matchLabels:
+//       machine.openshift.io/cluster-api-cluster: <cluster_name> <2>
+//       machine.openshift.io/cluster-api-machine-role: <label> <3>
+//       machine.openshift.io/cluster-api-machine-type: <label> <3>
+//       machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<AWS-zone> <4>
+// ----
+
 .MachineHealthCheck
 [source,yaml]
 ----
-apiVersion: healthchecking.openshift.io/v1alpha1
+apiVersion: machine.openshift.io/v1beta1
 kind: MachineHealthCheck
 metadata:
- name: example <1>
- namespace: openshift-machine-api
-Spec:
-  Selector:
+  name: example <1>
+  namespace: openshift-machine-api
+spec:
+  selector:
     matchLabels:
-      machine.openshift.io/cluster-api-cluster: <cluster_name> <2>
-      machine.openshift.io/cluster-api-machine-role: <label> <3>
-      machine.openshift.io/cluster-api-machine-type: <label> <3>
-      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<AWS-zone> <4>
+      machine.openshift.io/cluster-api-cluster: <infrastructureID> <2>
+      machine.openshift.io/cluster-api-machine-role: <role> <3>
+      machine.openshift.io/cluster-api-machine-type: <role> <3>
+      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<zone> <4>
+      role: worker <5>
```

Comment:
`role: worker` this is not a thing, can be removed. To clarify the matchLabels is just an example, may be make that clear in the text as well.

Reply:
Thanks for clarifying; I'll make that clear.